content/lessons/02_ownership/index.md

```diff
@@ -60,14 +73,17 @@ If you recall the RAII (Resource Acquisition Is Initialization) pattern from C++
 Let's expand on the last example. The scoping is not really important for that one, so we don't include it here.
 
 ```rust
-let s = String::from("a string"); // same thing, 's' is now an owner
+// Same thing, 's' is now an owner.
+let s = String::from("a string");
 
-let s2 = s; // easy, 's2' becomes another owner... right?
+// Easy, 's2' becomes another owner... right?
+let s2 = s;
 
-println!("And the contents are: {}", s); // this doesn't work, can you guess why?
+// This doesn't work, can you guess why?
+println!("And the contents are: {}", s);
 ```
 
-At first glance everything looks great. If we write this code (well, an equivalent of it) in basically any other popular language, it will compile no issue - but it does not here and there's a good reason why.
+At first glance everything looks great. If we write this code (well, an equivalent of it) in basically any other popular language, it will compile no issue - but it does *not* compile here and there's a good reason why.
```

Comment:
```suggestion
At first glance everything looks great. If we write this code (well, an equivalent of it) in basically any other popular language, it will compile without issues - but it does *not* compile here and there's a good reason why.
```